### PR TITLE
[CBRD-23576] Translate ro_Ro message

### PR DESCRIPTION
**Pull request by ribram**
http://jira.cubrid.org/browse/CBRD-23576

### CODE REVIEW

**Comment by eseokoh** [comment]
I don't know why loaddb msg 116, 117 and 118 remain untranslated.

File: msg/ro_RO.utf8/cubrid.msg [hunk 1]
```diff
@@ -1267,22 +1267,22 @@ $ LOADDB
...
+1201 NULL nu poate fi folosit pentru numele unei chei dintr-un JSON de tip Obiect.
+1202 Calea "%1$s" nu exista in %2$s.
+1203 Tipul JSON-ului de la calea %1$s este invalid. Se așteaptă %2$s, dar s-a găsit %3$s.
 1204 Cheia "%1$s" din json este duplicata
-1205 Invalid type %1$s. Expecting JSON document compatible data type (STRING or JSON).
-1206 Calea Json "%1$s" nu adresează un element al unui Json ARRAY.
-1207 Cale de JSON invalida. Index is larger than allowed by system parameter json_max_array_idx.
-1208 Invalid one/all argument. The one/all argument may only take one of these two values: "one", "all".
+1205 Tipul %1$s este invalid. Se așteaptă un tip compatibil cu un document JSON (STRING sau JSON).
+1206 Calea Json "%1$s" nu adresează un element al unui Json de tip Vector.
```
Comment (razvanradulescu) [inline msg/ro_RO.utf8/cubrid.msg]:
```suggestion
1206 Calea JSON "%1$s" nu adresează un element al unui JSON de tip ARRAY.
```

File: msg/ro_RO.utf8/cubrid.msg [hunk 2]
```diff
@@ -1267,22 +1267,22 @@ $ LOADDB
...
+1207 Cale de JSON invalida. Indexul este mai mare decât valoarea permisă de parametrul de sistem "json_max_array_idx".
```
Comment (razvanradulescu) [inline msg/ro_RO.utf8/cubrid.msg]: Should we put the name of system parameter as an argument to the message catalog ?

File: msg/ro_RO.utf8/utils.msg [hunk 3]
```diff
@@ -452,12 +452,12 @@ $set 12 MSGCAT_UTIL_SET_LOADDB
...
 115 una sau mai multe erori la încărcarea fişierului de intrare de control
-116 Total %8d object(s) inserted, %d object(s) failed.\n
-117 Load failed.\n
-118 Maximum length of class name is %1$d bytes.\n
-119 Total %1$8d object(s) checked, %2$d object(s) failed.\n
-120 data file does not contain '%%class' specifier or '--table' option is missing\n
-121 Class %1$s is ignored.\n
+116 Total %8d obiect(e) inserate, %d obiect(e) fără success.\n
+117 Încărcarea a eşuat.\n
+118 Lugimea maximă permisă pentru numele clasei este de %1$d octeţi.\n
```
Comment (razvanradulescu) [inline msg/ro_RO.utf8/utils.msg]:
```suggestion
118 Lungimea maximă permisă pentru numele clasei este de %1$d octeţi.\n
```

File: msg/ro_RO.utf8/utils.msg [hunk 4]
```diff
@@ -452,12 +452,12 @@ $set 12 MSGCAT_UTIL_SET_LOADDB
...
-117 Load failed.\n
-118 Maximum length of class name is %1$d bytes.\n
-119 Total %1$8d object(s) checked, %2$d object(s) failed.\n
-120 data file does not contain '%%class' specifier or '--table' option is missing\n
-121 Class %1$s is ignored.\n
+116 Total %8d obiect(e) inserate, %d obiect(e) fără success.\n
+117 Încărcarea a eşuat.\n
+118 Lungimea maximă permisă pentru numele clasei este de %1$d octeţi.\n
+119 Total %1$8d obiect(e) verificate, %2$d obiect(e) fără success.\n
+120 Fişierul de date nu conţine specificatorul '%%class' şi opţiunea '--table' lispeşte\n
```
Comment (razvanradulescu) [inline msg/ro_RO.utf8/utils.msg]:
```suggestion
120 Fişierul de date nu conţine specificatorul '%%class' sau opţiunea '--table' lispeşte\n
```

Comment (ribram) [reply msg/ro_RO.utf8/utils.msg]: Actually, I think the English error should be changed to "and".

File: msg/ro_RO.utf8/cubrid.msg (hunk 2 above)
Comment (ribram) [reply msg/ro_RO.utf8/cubrid.msg]: We could do that, yes.

**Pull request merged**